### PR TITLE
upgrade react native to 0.60.5

### PR DESCRIPTION
**Pull request by okenshields**
## Summary

Upgrade react native version to 0.60.5

## Related issues

N/A

## Changelog

* message 
Upgrade react native version to 0.60.5

### CODE REVIEW

File: react-native-in-app-payments-quickstart/ios/Podfile.lock [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    - React-Core (= 0.60.5)
+  - React-RCTText (0.60.5):
+    - React-Core (= 0.60.5)
+  - React-RCTVibration (0.60.5):
+    - React-Core (= 0.60.5)
+  - React-RCTWebSocket (0.60.5):
+    - React-Core (= 0.60.5)
+  - RNGestureHandler (1.4.1):
+    - React
+  - RNSquareInAppPayments (1.1.0):
```
Comment (hukid) [inline react-native-in-app-payments-quickstart/ios/Podfile.lock]: is this the right version? I though you've updated the version of plugin.

File: react-native-in-app-payments-quickstart/android/app/src/debug/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (okenshields) [reply react-native-in-app-payments-quickstart/android/app/src/debug/AndroidManifest.xml]: This change was from the upgrader tool but look likes it's not required at least not in my testing. Let me remove

File: react-native-in-app-payments-quickstart/android/build.gradle [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
+        buildToolsVersion = "28.0.3"
         minSdkVersion = 21
-        compileSdkVersion = 27
-        targetSdkVersion = 27
-        supportLibVersion = "27.1.1"
-        sqipVersion="1.1.0"
+        compileSdkVersion = 28
+        targetSdkVersion = 28
+        supportLibVersion = "28.0.0"
+        sqipVersion="1.2.0"
```
[the remark this answers is not in the document]
Comment (okenshields) [reply react-native-in-app-payments-quickstart/android/build.gradle]: Was testing both. but let me change back to 1.1.0 for me.

File: react-native-in-app-payments-quickstart/ios/Podfile.lock (hunk 1 above)
Comment (okenshields) [reply react-native-in-app-payments-quickstart/ios/Podfile.lock]: Let me update the podspec to reflect the change. Both should work as the plugin change doesn't really affect ios.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### v1.2.2 September 6, 2019
+
+* Upgraded to Android SDK 28
```
[the remark this answers is not in the document]
Comment (okenshields) [reply CHANGELOG.md]: Updated.

File: package.json [hunk 5]
```diff
@@ -34,11 +34,11 @@
     "jest": "24.1.0",
     "metro-react-native-babel-preset": "0.51.1",
     "react": "16.6.3",
-    "react-native": "0.57.8",
+    "react-native": "0.60.5",
```
[the remark this answers is not in the document]
Comment (okenshields) [reply package.json]: Seems to be ok. One concern is that our example app is using specific version so we are not sure if the plugin works with all the version... I assume we will just fix issues if found.

[the remark this answers is not in the document]
Comment (hukid) [reply package.json]: SG!

**Pull request merged**